core/server/proxy/src/main/java/alluxio/proxy/s3/signature/AuthorizationV2HeaderParser.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.proxy.s3.signature;
+
+import alluxio.proxy.s3.S3Exception;
+import alluxio.proxy.s3.S3ErrorCode;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Class to parse V2 auth information from header.
+ */
+public class AuthorizationV2HeaderParser implements SignatureParser {
+
+  private final String mAuthHeader;
+
+  /**
+   * @param authHeader authorization header
+   */
+  public AuthorizationV2HeaderParser(String authHeader) {
+    mAuthHeader = authHeader;
+  }
+
+  /**
+   * This method parses the authorization header.
+   */
+  @Override
+  public SignatureInfo parseSignature() throws S3Exception {
+    if (mAuthHeader == null || !mAuthHeader.startsWith(SignerConstants.IDENTIFIER + " ")) {
```

Comment:
Also, if this `IDENTIFIER` is only used here, define it in this class as a private variable, and add the empty space in it rather than do an additional string concatenation.